pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -120,6 +120,9 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 				if !ok {
 					return nil, fmt.Errorf("plugin %v does not extend score plugin", sc.Name)
 				}
+				if f.pluginNameToWeightMap[p.Name()] == 0 {
+					return nil, fmt.Errorf("score plugin %v is not configured with weight", p.Name())
```

Comment:
I wonder if the check on line 98 should handle 0 value similarly (returning error):
```
		if f.pluginNameToWeightMap[name] == 0 {
```

Reply:
The [comment](https://github.com/kubernetes/kubernetes/pull/80305#discussion_r304919930) explained it.